### PR TITLE
add query for windows uptime

### PR DESCRIPTION
**Pull request by brianbaker**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #1103, #1104 

### The problem
The changes in #1104 improved the metrics displayed for windows hosts, however, a separate query is needed to get the uptime of windows hosts (`[[inputs.system]]` is not implemented for windows).

### The Solution
Add a query to to the hosts page to pull the uptime from win perf counters.

### CODE REVIEW

File: ui/src/hosts/apis/index.js [hunk 1]
```diff
@@ -5,17 +5,25 @@ import _ from 'lodash'
...
+    query: [
+      'select mean(usage_user) from cpu where cpu = \'cpu-total\' and time > now() - 10m group by host',
+      'select mean("load1") from "system" where time > now() - 10m group by host',
+      'select non_negative_derivative(mean(uptime)) as deltaUptime from "system" where time > now() - 10m group by host, time(1m) fill(0)',
+      'select mean("Percent_Processor_Time") from win_cpu where time > now() - 10m group by host',
+      'select mean("Processor_Queue_Length") from win_system where time > now() - 10s group by host',
+      'select non_negative_derivative(mean("System_Up_Time")) as deltaUptime from "telegraf"."autogen"."win_uptime" where time > now() - 10m group by host, time(1m) fill(0)',
+      'show tag values from /win_system|system/ with key = "host"'].join('; '),
```
Comment (121watts) [inline ui/src/hosts/apis/index.js]: Thanks for cleaning this up!

**Comment by 121watts** [comment]
Thanks for the PR!  LGTM 🚢

**Comment by danielnelson** [comment]
@brianbaker Can you open a [new issue](https://github.com/influxdata/telegraf/issues/new) on Telegraf to add uptime support on Windows?

**Comment by brianbaker** [comment]
Opened https://github.com/influxdata/telegraf/issues/2738

**Comment by cryptoquick** [comment]
@brianbaker Thanks for the contribution! A small update, I'm nearly finished with making Chronograf able to build on Windows. Once I merge that work into master, I can then pull that into this branch and actually be able to test this commit properly! I appreciate your patience.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
   1. [#1337](https://github.com/influxdata/chronograf/pull/1337): Add support for blank hostnames on the Host List page
   1. [#1340](https://github.com/influxdata/chronograf/pull/1340): Fix case where the Explorer and cell editor falsely assumed there was no active query
   1. [#1338](https://github.com/influxdata/chronograf/pull/1338): Require url and name when adding a new source
+  1. [#1349](https://github.com/influxdata/chronograf/pull/1349): Add query for windows uptime
```
Comment (cryptoquick) [inline CHANGELOG.md]: This needs to be updated so the change is not in the section for the last release.

File: ui/src/hosts/apis/index.js (hunk 1 above)
Comment (cryptoquick) [inline ui/src/hosts/apis/index.js]: I'd prefer this to be a multi-line string, with backticks. You can also remove the escaping once that's done.

File: ui/src/hosts/apis/index.js [hunk 3]
```diff
@@ -5,17 +5,25 @@ import _ from 'lodash'
...
     const loadSeries = _.get(resp, ['data', 'results', '1', 'series'], [])
-    const winCPUSeries = _.get(resp, ['data', 'results', '2', 'series'], [])
-    const winLoadSeries = _.get(resp, ['data', 'results', '3', 'series'], [])
-    const uptimeSeries = _.get(resp, ['data', 'results', '4', 'series'], [])
-    const allHostsSeries = _.get(resp, ['data', 'results', '5', 'series'], [])
+    const uptimeSeries = _.get(resp, ['data', 'results', '2', 'series'], [])
+    const winCPUSeries = _.get(resp, ['data', 'results', '3', 'series'], [])
+    const winLoadSeries = _.get(resp, ['data', 'results', '4', 'series'], [])
+    const winUptimeSeries = _.get(resp, ['data', 'results', '5', 'series'], [])
+    const allHostsSeries = _.get(resp, ['data', 'results', '6', 'series'], [])
```
Comment (cryptoquick) [inline ui/src/hosts/apis/index.js]: Is there any rhyme or reason for this order? @121watts

**Comment by brianbaker** [comment]
Hi All - sorry been away for several days.  Do you need anything else from me on this PR or are you guys running with it?

**Comment by cryptoquick** [comment]
not quite yet, @brianbaker 
if you can address my comments, that'd be helpful.

**Comment by cryptoquick** [comment]
Perfect. Looks great. Thank you.

**Pull request merged**